sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/RulesImpl.java

```diff
@@ -0,0 +1,169 @@
+                @HeaderParam("If-Match") String ifMatch,
+                @BodyParam("application/xml") Object requestBody,
+                Context context);
+
+        @Delete("/{topicName}/subscriptions/{subscriptionName}/rules/{ruleName}")
+        @ExpectedResponses({200})
+        @UnexpectedResponseExceptionType(ServiceBusManagementErrorException.class)
+        Mono<Response<Object>> delete(
+                @HostParam("endpoint") String endpoint,
+                @PathParam("topicName") String topicName,
+                @PathParam("subscriptionName") String subscriptionName,
+                @PathParam("ruleName") String ruleName,
+                @QueryParam("api-version") String apiVersion,
+                Context context);
+    }
+
+    /**
+     * Get the details about the rule of a subscription of a topic.
+     *
+     * @param topicName name of the topic.
+     * @param subscriptionName name of the subscription.
+     * @param ruleName name of the filter.
+     * @param enrich A query parameter that sets enrich to true or false.
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ServiceBusManagementErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the details about the rule of a subscription of a topic.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Object>> getWithResponseAsync(
+            String topicName, String subscriptionName, String ruleName, Boolean enrich, Context context) {
+        return service.get(
+                this.client.getEndpoint(),
+                topicName,
+                subscriptionName,
+                ruleName,
+                enrich,
+                this.client.getApiVersion(),
+                context);
+    }
+
+    /**
+     * Create or update a rule.
+     *
+     * @param topicName name of the topic.
+     * @param subscriptionName name of the subscription.
+     * @param ruleName name of the filter.
+     * @param requestBody Any object.
+     * @param ifMatch Match condition for an entity to be updated. If specified and a matching entity is not found, an
+     *     error will be raised. To force an unconditional update, set to the wildcard character (*). If not specified,
+     *     an insert will be performed when no existing entity is found to update and a replace will be performed if an
+     *     existing entity is found.
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ServiceBusManagementErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return any object.
```

Comment:
Should we indicate what this object will contain , when user create/update rule what kind of information will come back ? Same comment for other similar `return any object` .

Reply:
This is in implementation class. It is autogenerated.